demo/src/app/components/datepicker/demos/basic/datepicker-basic.ts

```diff
@@ -0,0 +1,19 @@
+import {Component} from '@angular/core';
+
+const now = new Date();
+
+@Component({
+  selector: 'ngbd-datepicker-basic',
+  template: require('./datepicker-basic.html')
+})
+export class NgbdDatepickerBasic {
+
+  model;
+
+  minDate = {year: 2000, month: 0, date: 7};
```

Comment:
Shouldn't it be `day` instead of `date`? For me `date` indicated the whole object. Could we please rename it?